receiver/prometheusreceiver/README.md

```diff
@@ -23,14 +23,18 @@ receivers:
     prometheus:
       config:
         scrape_configs:
-          - job_name: 'opencensus_service'
+          - job_name: 'otel-collector'
             scrape_interval: 5s
             static_configs:
-              - targets: ['0.0.0.0:8889']
-          - job_name: 'jdbc_apps'
-            scrape_interval: 3s
-            static_configs:
-              - targets: ['0.0.0.0:9777']
+              - targets: ['0.0.0.0:8888']
+          - job_name: k8s
+            kubernetes_sd_configs:
+            - role: pod
+            relabel_configs:
+            - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_scrape]
```

Comment:
Because people really care about this, maybe extend the text above to say "This receiver is a drop-in replacement for getting Prometheus to scrape your services. It supports the full set of Prometheus configuration, including service-discovery. Just like ..."

Reply:
Done, and also added `metric_relabel_configs` to the example.

Reply:
Sorry, I didn't realize that you suggest to expand phrasing in a different place (https://github.com/open-telemetry/opentelemetry-collector/blame/master/receiver/prometheusreceiver/README.md#L6), will fix.

Reply:
Fixed.